src/datepicker/Datepicker.vue

```diff
@@ -248,4 +261,10 @@ export default defineComponent({
 
   position: relative;
 }
+
+.clearable {
```

Comment:
CSS in this exact component is not scoped, so specifying such a generic name here without any nesting will surely lead to problems for other users of this component.